### PR TITLE
fix: update `get-token` api to return url as well.

### PR DESCRIPTION
**Pull request by rafiu007**

### CODE REVIEW

File: bayesian/api/api_v2.py [hunk 1]
```diff
@@ -68,7 +68,8 @@ def get_token():
     try:
         # return default key
         THREESCALE_PREMIUM_USER_KEY = os.getenv('THREESCALE_PREMIUM_USER_KEY')
-        return jsonify({"key": THREESCALE_PREMIUM_USER_KEY}), 200
+        PREMIUM_URL = os.getenv('PREMIUM_URL')
+        return jsonify({"url": PREMIUM_URL, "key": THREESCALE_PREMIUM_USER_KEY}), 200
```
Comment (yzainee) [inline bayesian/api/api_v2.py:72]: corresponding changes in the unit tests should be done as well.

**Review by yzainee — CHANGES_REQUESTED** [review]

File: bayesian/api/api_v2.py [hunk 2]
```diff
@@ -68,7 +68,8 @@ def get_token():
     try:
         # return default key
         THREESCALE_PREMIUM_USER_KEY = os.getenv('THREESCALE_PREMIUM_USER_KEY')
-        return jsonify({"key": THREESCALE_PREMIUM_USER_KEY}), 200
+        PREMIUM_URL = os.getenv('PREMIUM_URL')
```
Comment (yzainee) [inline bayesian/api/api_v2.py:71]: PREMIUM_URL needs to be added to the template file

**Review by yzainee — CHANGES_REQUESTED** [review]

File: bayesian/api/api_v2.py (hunk 1 above)
Comment (rafiu007) [reply bayesian/api/api_v2.py:72]: Yes.On it.

File: bayesian/api/api_v2.py (hunk 2 above)
Comment (rafiu007) [reply bayesian/api/api_v2.py:71]: IT is added @yzainee

Comment (rafiu007) [reply bayesian/api/api_v2.py:71]: which template are you talking about?

Comment (yzainee) [reply bayesian/api/api_v2.py:71]: openshift/template.yaml

Comment (yzainee) [reply bayesian/api/api_v2.py:71]: oh i didnt notice.

File: openshift/template.yaml [hunk 3]
```diff
@@ -127,6 +127,11 @@ objects:
               secretKeyRef:
                 name: 3scale
                 key: three_scale_premium_user_key
+          - name: PREMIUM_URL
+            valueFrom:
+              secretKeyRef:
```
Comment (yzainee) [inline openshift/template.yaml:132]: if you are introducing a new secret, you will need this secret in the vault before you merge this PR

Comment (yzainee) [reply openshift/template.yaml:132]: I dont think this needs to be a secret. It can be a configurable value in app interface.

Comment (rafiu007) [reply openshift/template.yaml:132]: Since the token is a secret i made this also a secret.

File: bayesian/api/api_v2.py (hunk 1 above)
Comment (rafiu007) [reply bayesian/api/api_v2.py:72]: I feel there is no modicfication required in the tests. The response code test is already done.

Comment (yzainee) [reply bayesian/api/api_v2.py:72]: add the new key "url" in the test

File: openshift/template.yaml (hunk 3 above)
Comment (yzainee) [reply openshift/template.yaml:132]: there is no need to have this as a secret. please change it

Comment (rafiu007) [reply openshift/template.yaml]: Done.

**Review by yzainee — APPROVED** [review]

File: bayesian/api/api_v2.py (hunk 1 above)
Comment (rafiu007) [reply bayesian/api/api_v2.py:72]: Done

**Pull request merged**